docs/developer/app-store/apps/adyen/overview.mdx

```diff
@@ -37,3 +37,21 @@ Adyen App uses Adyen [Checkout API v70](https://docs.adyen.com/api-explorer/#/Ch
 :::
 
 If you want to self-host the Adyen app, reach out to [our team <Mail size={12}/>](mailto:hello@saleor.io).
+
+## Assumptions
+
+- App will return `CHARGE_ACTION_REQUIRED` or `AUTHORIZATION_ACTION_REQUIRED` as the transaction result when the response from Adyen to [initialize transaction session webhook](/developer/extending/webhooks/synchronous-events/transaction#initialize-transaction-session) or [process transaction session webhook](/developer/extending/webhooks/synchronous-events/transaction#process-transaction-session) webhooks does not include `pspReference` or `action`. This is because Saleor don't require `pspReference` in app response for this transaction result.
```

Comment:
"response from Adeyen to ... webhooks"

Its hard from the congnitive perspective. Can you make the order clear, so:
1. For requests X and Y
2. Adyen respond without pspReference and action
3. Will return A or B

Not this paragraph starts with last step, its hard to understand